packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/barriers-list.jsx

```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const BarriersList = ({
+    active_item_class_name,
+    base_class_name,
+    chosen_item,
```

Comment:
```suggestion
    selected_item,
```
@kate-deriv just a nit-pick😀